soundata/datasets/urbansound8k.py

```diff
@@ -0,0 +1,355 @@
+
+    .. code-block:: latex
+        J. Salamon, C. Jacoby and J. P. Bello, "A Dataset and Taxonomy for Urban Sound Research", 
+        22nd ACM International Conference on Multimedia, Orlando USA, Nov. 2014.
+
+    The creation of this dataset was supported by a seed grant by NYU's Center for Urban Science and Progress (CUSP).
+
+
+    Conditions of Use
+    -----------------
+
+    Dataset compiled by Justin Salamon, Christopher Jacoby and Juan Pablo Bello. All files are excerpts of recordings
+    uploaded to www.freesound.org. Please see FREESOUNDCREDITS.txt for an attribution list.
+    
+    The UrbanSound8K dataset is offered free of charge for non-commercial use only under the terms of the Creative Commons
+    Attribution Noncommercial License (by-nc), version 3.0: http://creativecommons.org/licenses/by-nc/3.0/
+    
+    The dataset and its contents are made available on an "as is" basis and without warranties of any kind, including 
+    without limitation satisfactory quality and conformity, merchantability, fitness for a particular purpose, accuracy or 
+    completeness, or absence of errors. Subject to any liability that may not be excluded or limited by law, NYU is not 
+    liable for, and expressly excludes, all liability for loss or damage however and whenever caused to anyone by any use of
+    the UrbanSound8K dataset or any part of it.
+
+
+    Feedback
+    --------
+
+    Please help us improve UrbanSound8K by sending your feedback to: justin.salamon@nyu.edu
+    In case of a problem report please include as many details as possible.
+
+"""
+
+import os
+from typing import BinaryIO, Optional, TextIO, Tuple
+
+import librosa
+import numpy as np
+import csv
+
+from soundata import download_utils
+from soundata import jams_utils
+from soundata import core
+from soundata import annotations
+from soundata import io
+
+
+BIBTEX = """
+@inproceedings{Salamon:UrbanSound:ACMMM:14,
+	Address = {Orlando, FL, USA},
+	Author = {Salamon, J. and Jacoby, C. and Bello, J. P.},
+	Booktitle = {22nd {ACM} International Conference on Multimedia (ACM-MM'14)},
+	Month = {Nov.},
+	Pages = {1041--1044},
+	Title = {A Dataset and Taxonomy for Urban Sound Research},
+	Year = {2014}}
+"""
+REMOTES = {
+    "all": download_utils.RemoteFileMetadata(
+        filename="UrbanSound8K.tar.gz",
+        url="https://zenodo.org/record/1203745/files/UrbanSound8K.tar.gz",
+        checksum="9aa69802bbf37fb986f71ec1483a196e",
+        destination_dir="urbansound8k",
+    )
+}
+
+LICENSE_INFO = "Creative Commons Attribution Non Commercial 4.0 International"
+
+
+class Clip(core.Clip):
+    """urbansound8k Clip class
+
+    Args:
+        clip_id (str): clip id of the clip
+
+    Attributes:
+        audio_path (str): path to the audio file
+        slice_file_name (str): The name of the audio file. The name takes the following format: [fsID]-[classID]-[occurrenceID]-[sliceID].wav, where:
```

Comment:
This feels a bit long here, and since the info is already in the docs I'd simplify it

Reply:
Done, replaced everything after the first line with: "Please see the Dataset Info in the soundata documentation for further details."